core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -514,7 +514,7 @@ object KafkaConfig {
   val SocketReceiveBufferBytesDoc = "The SO_RCVBUF buffer of the socket sever sockets. If the value is -1, the OS default will be used."
   val SocketRequestMaxBytesDoc = "The maximum number of bytes in a socket request"
   val MaxConnectionsPerIpDoc = "The maximum number of connections we allow from each ip address"
```

Comment:
While we're at it, do you think it's worth mentioning that this can be set to 0 if there are overrides? And now that I'm thinking about it, maybe we missed this validation in the other patch?

Reply:
@hachikuji  Thanks for the review. Updated the doc and added a validation check.
Pls have a look.